spec/helpers/articles_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the ArticlesHelper. For example:
+#
+# describe ArticlesHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe ArticlesHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
Consider removing unused specs like articles_helper_spec.rb. 

To avoid adding these unwanted files to the repo in future, before running `git add .`, run `git status` and delete (`rm path/to/unwanted_file`) any files that you don't want to keep.